backend/apps/content/views.py

```
@@ -498,3 +501,41 @@ def get(self, request, lesson_slug):
                 {"error": "No feedback found for this lesson"},
                 status=status.HTTP_404_NOT_FOUND,
             )
+
+
+class ModuleDraftViewSet(viewsets.ModelViewSet):
+    queryset = ModuleDraft.objects.prefetch_related("lessons__quizzes").all()
+    serializer_class = ModuleDraftSerializer
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)